### ISSUE TITLE
[dev.icinga.com #13185] Check now won't refresh the status page

### ISSUE THREAD

**Issue by icinga-migration** [Icinga/icingaweb2#2620]
This issue has been migrated from Redmine: https://dev.icinga.com/issues/13185

**Created by knoth on 2016-11-14 10:53:39 +00:00**

Assignee: _elippmann_
Status: _Assigned_ 
Target Version: _(none)_
Last Update: _2016-11-17 09:57:45 +00:00 (in Redmine)_

---

To reproduce this bug select a Check in Icingaweb2. Now change the status of this check and execute "Check now".
On the left side (overview page) the check will refresh but on the right side (detailed status page) the status won't refresh automatically.
The page will refresh only after running an action like refresh or "Check now" again.

**Comment by lippserd** [comment]
Should be fixed. If you still have problems, please open a new issue.
